### ISSUE TITLE
inline display of options possible?

### ISSUE THREAD

**Issue by mtness** [jannicz/appointment-picker#34]
Hi there, 
I would like to display only the options and hide the input field - would this be possible?

Thank you and best regards, MT

**Comment by jannicz** [comment]
Yes, this should be easily possible. Apply some hidden styles on the input via CSS, i.e. `visibility: hidden` and the open the picker programmatically via the [API function](https://jannicz.github.io/appointment-picker/example/exposed-functions.html): `picker.open()`

You can additionally [change the template](https://jannicz.github.io/appointment-picker/example/render-on-init.html) for the markup you you need another HTML.

Tell me if that worked for you

**Comment by mtness** [comment]
Ok cool, will try!
While implementing it, another question arose, (#35 )

Thank you!
